msc/static/js/AddressCopy.js

```diff
@@ -0,0 +1,19 @@
+function FillBilling(f) {
```

Comment:
Good point.  I thought about that but considered this to be generically
reusable by any page that had addresses to clone (mailing, shipping,
physical)
On Nov 6, 2015 20:22, "Dan Nascimbeni" <notifications@github.com> wrote:

> In msc/static/js/AddressCopy.js
> <https://github.com/dnascimb/msc/pull/8#discussion_r44204599>:
>
> > @@ -0,0 +1,19 @@
> > +function FillBilling(f) {
>
> We didn't discuss this before but probably a good practice to make any of
> the JS files that correspond to a specific view, the same name as the view.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dnascimb/msc/pull/8/files#r44204599>.
>